main.go

```diff
@@ -215,6 +215,32 @@ func handleCheckSuite(gh *GithubClient, cs *CheckSuiteWebhook) error {
 		return nil
 	}
 
+
+	eventIsFromSupportedApp := false
+	for _, app := range gh.AppTargets {
```

Comment:
I think I'm missing something where does gh.AppTargets come from? How does it represent the see of apps we care about?

Reply:
It gets set on instantiation here: https://github.com/Azure/azure-sdk-actions/blob/main/main.go#L62. The default is `Azure Pipelines` and `Github Actions` as valid app targets. So any event or check suite status not created by one of those app identities will get skipped.

Reply:
If I'm understanding the code correctly isn't this "Azure Pipelines" and "GitHub Actions"? If so won't this still have us hitting this for the event processor?

Reply:
@weshaggard It won't, because the event processor never triggers this workflow, due to the github actions default behavior that workflows can't trigger workflows (only events allowlisted for `workflow_dispatch` will trigger: https://github.com/Azure/azure-dev/blob/main/.github/workflows/event.yml#L15).

The core issue here was that `Microsoft Github Policy Service` still triggered an _evaluation_ even if we ignored it from the evaluated check suite results. Now the only things that can trigger evaluation are pipelines/actions we care about for CI, which guarantees that there will always be at least one CI gate status in the check_suites response for us to evaluate.